pom.xml

```diff
@@ -239,6 +239,11 @@
         <artifactId>avaje-jex-ssl</artifactId>
         <version>3.4-RC2</version>
       </dependency>
+      <dependency>
+        <groupId>io.avaje</groupId>
+        <artifactId>avaje-jex-http3-flupke</artifactId>
+        <version>3.4-RC2</version>
```

Comment:
```suggestion
        <version>0.1</version>
```

Reply:
oh ye wrong version 🤦‍♂️ 

But keep this open I think we should wait until it's non-alpha?